unraid-ui/stories/components/common/Loading.stories.ts

```
@@ -1,10 +1,10 @@
-import type { Meta, StoryObj } from "@storybook/vue3";
-import Bar from "../../../src/components/common/loading/Bar.vue";
-import Error from "../../../src/components/common/loading/Error.vue";
-import Spinner from "../../../src/components/common/loading/Spinner.vue";
+import type { Meta, StoryObj } from '@storybook/vue3';
+import Bar from '../../../src/components/common/loading/Bar.vue';
+import Error from '../../../src/components/common/loading/Error.vue';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)